python/semantic_kernel/connectors/memory/postgres/postgres_memory_store.py

```diff
@@ -0,0 +1,335 @@
+import atexit
+import json
+import os
+from typing import List, Tuple
+
+import numpy
+from numpy import ndarray
+from psycopg import Cursor
+from psycopg_pool import ConnectionPool
+
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+
+# On Windows, Psycopg is not compatible with the default ProactorEventLoop.
+# Please use a different loop, for instance the SelectorEventLoop.
+# https://www.psycopg.org/psycopg3/docs/advanced/async.html
+if os.name == "nt":
+    import asyncio
+
+    asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+
+MEMORY_TABLE_NAME = "sk_memory_table"
```

Comment:
Makes sense, I had originally set this up to allow for delineation of the collections to be at the schema level. Since the direction is to go with table level delineation, I will make the necessary changes accordingly.